### DISCUSSION TITLE
Does swiftgen command supports Linux?

### DISCUSSION THREAD

**Discussion by wulinman** [SwiftGen/SwiftGen#974] [Ideas]
How can I install swiftgen command in Linux debian?

**Comment by djbe** [comment]
Not at this moment. We currently depend on a bunch of AppKit functionality for our parsers & generators.

Changing all of that to drop AppKit is "possible", but definitely not easy 😅 We have no interest in doing this ourselves at this point, but feel free to take a stab at it if you really want.
